tests/e2e/util.go

```diff
@@ -3152,3 +3153,26 @@ func statefulSetFromManifest(fileName string, ss *appsv1.StatefulSet) (*appsv1.S
 
 	return ss, nil
 }
+
+func waitForAllPodsToBeUpAndRunning(ctx context.Context, client clientset.Interface, namespace string) {
```

Comment:
fetch number of pods before restart and can use `fpod.WaitForPodsRunningReady()` instead